gtk/src/default/gtk-3.0/_colors.scss

```diff
@@ -88,3 +88,9 @@ $coloured_focus_border_color: transparentize(white, 0.3);
 // Yaru: Ignore transformations when using dark accent colors
 $progress_bg_color: $accent_bg_color;
 $checkradio_bg_color: $accent_bg_color;
+
+// Optimize accent-color definition for default background colors
+@import 'sass-utils';
+$yaru-accent-color: optimize-contrast($bg_color, $accent_bg_color);
+$link_color: if($variant =='light', $yaru-accent-color, $yaru-accent-color);
```

Comment:
I think the conditional function is not needed here (same for gtk4).

Reply:
indeed... -_-